client/my-sites/site-settings/form-security.jsx

```diff
@@ -92,9 +92,7 @@ class SiteSettingsFormSecurity extends Component {
 					</div>
 				) }
 
-				<SettingsSectionHeader
-					title={ translate( 'WordPress.com log in / single sign-on (SSO)' ) }
-				/>
+				<SettingsSectionHeader title={ translate( 'WordPress.com log in' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **27** times:
`translate( 'Log in to WordPress.com' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).